File: chapter-18-jpa-jooq/code-java/src/main/java/com/example/demo/repository/AuthorJpaRepository.java
package com.example.demo.repository;

import com.example.demo.entity.AuthorEntity;
import org.springframework.data.jpa.repository.JpaRepository;

/**
 * 【Chapter 18】 CQRS Command(CUD) 영역: Spring Data JPA
 */
public interface AuthorJpaRepository extends JpaRepository<AuthorEntity, Integer> {
}

File: chapter-18-jpa-jooq/code-kotlin/src/main/kotlin/com/example/demo/repository/AuthorJpaRepository.kt
package com.example.demo.repository

import com.example.demo.entity.AuthorEntity
import org.springframework.data.jpa.repository.JpaRepository

/**
 * 【Chapter 18】 CQRS Command(CUD) 영역: Kotlin Spring Data JPA
 */
interface AuthorJpaRepository : JpaRepository<AuthorEntity, Int>
